src/sql/workbench/contrib/notebook/browser/cellToolbarActions.ts

```diff
@@ -151,10 +151,11 @@ export class CellToggleMoreActions {
 
 	public onInit(elementRef: HTMLElement, context: CellContext) {
 		this._moreActionsElement = elementRef;
+		this._moreActionsElement.setAttribute('aria-haspopup', 'menu');
```

Comment:
Where are you getting 'menu' from for the value? All the examples I see use true/false